packages/pn-personagiuridica-webapp/src/component/DomicileBanner/DomicileBanner.tsx

```diff
@@ -36,10 +37,10 @@ const DomicileBanner = () => {
   };
 
   useEffect(() => {
-    if (legalDomicile && legalDomicile.length > 0) {
+    if ((legalDomicile && legalDomicile.length > 0) || isGroupAdmin ) {
```

Comment:
I sohuld revome the check here and I sohuld put the condition {isGroupAdmin && <DomicileBanner...} where the component is used